tests/Feature/UnitQueueTest.php

```diff
@@ -31,6 +32,10 @@ private function basicSetup(): void
         $this->planetSetObjectLevel('research_lab', 1);
         $this->playerSetResearchLevel('energy_technology', 1);
         $this->playerSetResearchLevel('combustion_drive', 1);
+
+        //refresh player
+        $playerService = app()->make(PlayerService::class, ['player_id' => $this->currentUserId]);
+        $playerService->load($this->currentUserId);
```

Comment:
I tried running the tests without this player refresh on my local machine and they still all pass. Could you check again if the tests still run when you remove this code?

Reply:
I did the same thing now and it works. I have no idea why it wasn't working before, perhaps the branch was outdate. So I removed this part.

Reply:
👍 Thanks for double-checking!